src/main/java/db/disk/dataHandler/DiskDataPosition.java

```diff
@@ -62,12 +75,12 @@ public int compareTo(DiskDataPosition o) {
 	@Override
 	public boolean equals(Object o) {
 		DiskDataPosition pos = (DiskDataPosition) o;
-		return pos.nodeID == this.nodeID && pos.fileID == this.fileID && pos.lineIndex == this.lineIndex;
+		return pos.fileID == this.fileID && pos.lineIndex == this.lineIndex; // pos.nodeID == this.nodeID && 
 	}
 
 	@Override
 	public int hashCode() {
-		return (int) Math.pow(31, nodeID) * this.fileID + this.lineIndex;
+		return (int) this.fileID + this.lineIndex; // Math.pow(31, nodeID) * 
```

Comment:
ça c'est bof. j'aurais quand même mis ça, histoire de ne pas avoir trop de collisions :
```suggestion
		return (int) 31 * this.fileID + this.lineIndex;
```